neverbleed.h

```diff
@@ -41,6 +41,14 @@ typedef struct st_neverbleed_t {
     unsigned char auth_token[NEVERBLEED_AUTH_TOKEN_SIZE];
 } neverbleed_t;
 
+#define ROUND2WORD(n) (n + 64 - 1 - (n + 64 - 1) % 64)
+#define BITMASK(b) (1 << ((b) % CHAR_BIT))
+#define BITBYTE(b) ((b) / CHAR_BIT)
+#define BITSET(a, b) ((a)[BITBYTE(b)] |= BITMASK(b))
+#define BITUNSET(a, b) ((a)[BITBYTE(b)] &= ~BITMASK(b))
+#define BITBYTES(nb) ((nb + CHAR_BIT - 1) / CHAR_BIT)
+#define BITCHECK(a, b) ((a)[BITBYTE(b)] & BITMASK(b))
```

Comment:
Can we move these to the source file?

I think we should, assuming that they will be used only in neverbleed.c. The other approach would be to add `neverbleed_` prefix to all the macros to make sure we avoid conflicts, but I do not think we need to keep them in a public header.

This is the only issue I would like to discuss before merging this PR. Please let me know what you think.

Reply:
Thank you for the review. Agreed, makes sense to move it out of the header file - esp with the high potential for collision in the naming.
Will push the changes in a bit. 👍